### ISSUE TITLE
Displayed font is always black in all dark themes

### ISSUE THREAD

**Issue by rcanand** [microsoft/vscode#152568]
Issue Type: <b>Bug</b>

Open any text file. Change theme to a dark theme - Github Dark, Abyss, any dark theme really. Notice that the text is black, even though all these dark themes use a light font color.

- This happened after a recent (automatic) update. 
- Light themes seem to work fine (but they are also showing black font, I think).
- This happens only within the text editor.

VS Code version: Code 1.68.1 (30d9c6cd9483b2cc586687151bcbcd635f373630, 2022-06-14T12:52:13.188Z)
OS version: Darwin x64 21.5.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7920HQ CPU @ 3.10GHz (8 x 3100)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.05GB free)|
|Process Argv|--crash-reporter-id 9bb3e187-b95e-446c-ac99-c5a4e1818807|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (39)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-django|bat|1.10.0
unique-lines|bib|1.0.0
format-json|Cle|1.0.2
permute-lines|ear|1.1.0
prettier-vscode|esb|9.5.0
gc-excelviewer|Gra|4.2.55
Python-autopep8|him|1.0.2
vscode-edit-csv|jan|0.7.0
vscode-3dviewer|md2|1.0.0
rainbow-csv|mec|2.4.0
vscode-antlr4|mik|2.3.0
theme-monokai-pro-vscode|mon|1.1.20
csharp|ms-|1.25.0
python|ms-|2022.8.0
vscode-pylance|ms-|2022.6.20
jupyter|ms-|2022.5.1001601848
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.8
cpptools|ms-|1.10.7
vscode-stl-viewer|mts|0.3.0
word-count|Oli|0.0.1
material-icon-theme|PKi|4.18.1
java|red|1.7.0
rust|rus|0.7.8
jinjahtml|sam|0.17.0
even-better-toml|tam|0.16.3
bootstrap4-vscode|the|6.1.0
corona-tools|Tob|0.2.1
ansible|tom|1.2.0
sort-lines|Tyr|1.9.1
vscode-java-debug|vsc|0.41.0
vscode-java-dependency|vsc|0.19.1
vscode-java-pack|vsc|0.23.0
vscode-java-test|vsc|0.35.0
vscode-icons|vsc|11.12.0
cors-browser|Wsc|1.0.11
markdown-pdf|yza|1.4.4
markdown-all-in-one|yzh|3.4.3
material-theme|zhu|3.15.0

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdf:30486550

```

</details>

**Comment by IllusionMH** [comment]
/needsMoreInfo

Are you using Source Code Pro or any other non-default font?
May be duplicate of #100840

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's missing some basic information or in some other way doesn't follow our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). Please take the time to review these and update the issue.

Happy Coding!

**Comment by rcanand** [comment]
> /needsMoreInfo
> 
> Are you using Source Code Pro or any other non-default font? May be duplicate of #100840

Thanks, that helped find and resolve the issue for me. Yes, I was using `Source Code Pro Light`, which caused this issue. I reinstalled Source Code Pro fonts from fonts.google.com, still had the same issue. I then changed it to `Source Code Pro ExtraLight` and it worked. I also confirmed that it was using ExtraLight by removing all other fonts. I think ExtraLight used to be much thinner before, it seems a bit heftier now, but works for now :-)
